src/js/views/globals/root_views.js

```diff
@@ -297,6 +307,7 @@ const RootView = CollectionView.extend({
     this.addRegionView('alert', new TopRegionView());
     this.addRegionView('modal', new ModalRegionView({ $body }));
     this.addRegionView('modalSmall', new ModalRegionView({ $body }));
+    this.addRegionView('modalSidebar', new ModalSidebarRegionView({ $body }));
```

Comment:
I believe the order here determines the stack placement.  Which is why we should not `z-index` these.  I think it's probably `modalSidebar` -> `modal` -> `modalSmall`